CMakeLists.txt

```diff
@@ -443,9 +443,9 @@ if(ENABLE_TESTING)
         # 2. Run the relevant tests for the part of the code you're interested in.
         #    For the reference coverage measurement, see
         #    tests/scripts/basic-build-test.sh
-        # 3. Run scripts/lcov.sh to generate an HTML report.
+        # 3. Run framework/scripts/lcov.sh to generate an HTML report.
         ADD_CUSTOM_TARGET(lcov
-            COMMAND scripts/lcov.sh
+            COMMAND framework/scripts/lcov.sh
```

Comment:
Despite my previous approval I think I would suggest a change here. Can we make the `framework` path a bit more generic using `MBEDTLS_FRAMEWORK_DIR` as you did in the `development` [branch PR](https://github.com/Mbed-TLS/mbedtls/pull/10450/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR492)?